test/integration/serverTest.js

```diff
@@ -976,4 +1009,85 @@ describe('Brave Sensor server', () => {
       })
     })
   })
+
+  describe('POST request heartbeat events with mock INS Firmware State Machine', () => {
+    beforeEach(async () => {
+      await redis.clearKeys()
+      await db.clearSessions()
+      await db.clearLocations()
+      await db.clearClients()
```

Comment:
Now that I've merged https://github.com/bravetechnologycoop/BraveSensor-Server/pull/128 , you can replace the `db.clear*` calls with `db.clearTables()` (less lines of code, less likely that a table will be forgotten, less likely that the tables will be cleared in the wrong order).